dapp-developer-guide/tutorials/use-ethstorage-sdk-to-upload-and-download-files.md

```diff
@@ -15,80 +15,85 @@ You can install `ethstorage-sdk` by the following command:
 npm i ethstorage-sdk
 ```
 
-## Step 2: Upload Files
+## Step 2: Manage Files
 
+### 2.1 Create Flat Directory
+
+In this section, you will create a `FlatDirectory` contract for managing files.
 
 ```js
-const {BlobUploader, EncodeBlobs} = require("ethstorage-sdk");
+const {EthStorage} = require("ethstorage-sdk");
 
-const upload = async (filePath) => {
+const create = async () => {
+    const ETH_STORAGE_ADDRESS = "0x804C520d3c084C805E37A35E90057Ac32831F96f";
     const rpc = "https://rpc.sepolia.org";
     const privateKey = "0x...";
-    const tx = {
-      value: cost,
-      to: to,
-      data: data,
-    };
-    
-    const content = fs.readFileSync(filePath);
-    const blobs = EncodeBlobs(content);
-    const blobUploader = new BlobUploader(rpc, privateKey);
-    const hash = await blobUploader.sendTx(tx, [blobs[0], blobs[1]...) // max is 6
-    // hash : 0x37df32c7a3c30d3...52453dadacc838461d8629016
-}
-```
-
-## Step 3: Manage Files
+ 
+    const ethStorage = new EthStorage(rpc, privateKey);
+    await ethStorage.deploy(ETH_STORAGE_ADDRESS);
+    // FlatDirectory address: 0x37df32c7a3c30d352453dadacc838461d8629016
 
-### 3.1 Create Flat Directory
+    or
 
-In this section, you will create a `FlatDirectory` contract for managing files.
+    // If you are on the Sepola network, you can do the following:
+    await ethStorage.deploySepolia();
+}
+```
 
+If you already have a `FlatDirectory` contract, you can set it directly.
 
 ```js
 const {EthStorage} = require("ethstorage-sdk");
 
 const create = async () => {
-    const ETH_STORAGE_ADDRESS = "0x804C520d3c084C805E37A35E90057Ac32831F96f";
     const rpc = "https://rpc.sepolia.org";
     const privateKey = "0x...";
+    const flatDirectory = "0x37df32c7a3c30d352453dadacc838461d8629016";
  
-    const ethStorage = new EthStorage(rpc, privateKey);
-    await ethStorage.deployBlobDirectory(ETH_STORAGE_ADDRESS);
-    // flatDirectory address: 0x37df32c7a3c30d352453dadacc838461d8629016
+    const ethStorage = new EthStorage(rpc, privateKey, flatDirectory);
 }
 ```
 
-### 3.2: Upload Files To Flat Directory
+
+### 2.2: Upload Files To Flat Directory
 
 In this section, you will upload some files into the `FlatDirectory` that you just created.
 
 ```js
 const {EthStorage} = require("ethstorage-sdk");
 
 const upload = async (filePath) => {
-    const rpc = "https://rpc.sepolia.org";
-    const privateKey = "0x...";
- 
-    const ethStorage = new EthStorage(rpc, privateKey);
-    await ethStorage.upload(filePath);
+    // You can set the path of a file or folder. If using a browser, you can specify the file object selected by the 'input'.
+    await ethStorage.upload(pathOrFile);
     // hash: 0x753...45836
+
+    or
```

Comment:
People like to copy and paste code directly, so split the code sample and remove  `or` .